docs/gettingUpdates.md

```diff
@@ -0,0 +1,98 @@
+# Getting updates
+
+(https://core.telegram.org/bots/api#getting-updates)
+
+Telegram offers two different ways to listen to bot's updates:
+
+* Using the `getUpdates` API operation. You can use this operation to proactively get updates from Telegram servers (bot implementations using this operation use to incorporate a polling mechanism to periodically ask for updates).  
+* Setting up a webhook. With this approach, the updates are directly pushed from Telegram servers to the server where our bot is deployed.
+
+Both methods cannot be used altogether, so you'll have to choose one of them depending on your needs.
+
+
+## Using the `getUpdates` API operation 
+
+Telegram exposes an API operation that let us query the latest updates correspondent to a given bot. For more details about this operation check out https://core.telegram.org/bots/api#getupdates
+
+To receive updates in a near to realtime way you'll have to implement a polling mechanism to periodically ask Telegram for new updates. 
+
+This library provides an API that lets you getting updates through the mentioned API operation with a built-in polling implementation.
+
+You'll just need to create an instance of a bot and call to the `startPolling` method: 
+
+```
+bot {
+    token = "YOUR_BOT_TOKEN"
+    // bot implementation
+}.startPolling()
+```
+
+For a more detailed example you can check the `dispatcher` and `echo` samples you can find in the `samples` folder of this project.
+
+
+## Setting up a webhook
+
+Telegram offers a way to receive updates through a webhook. You just have to indicate to Telegram the url where you want to receive the updates and Telegram will send an HTTP POST with updates every time updates are received for a given bot.
```

Comment:
I think Telegram sends HTTPS POST (https://core.telegram.org/bots/api#setwebhook)

Reply:
Yes, it's HTTP**S**. I'll update the doc to make it clearer.